Tests/CoreTests/GlobTests.swift

```diff
@@ -42,7 +42,7 @@ class GlobTests: XCTestCase {
     }
 
     private func newTmpDir() -> String {
-        var tmpDirTmpl = "/tmp/glob-test.XXXXX".cString(using: .utf8)!
+        var tmpDirTmpl = "/tmp/glob-test.XXXXXX".cString(using: .utf8)!
```

Comment:
Out of curiosity, what caused this change?

Reply:
This was a weird one! In glibc, `mkstemp` requires that [exactly six characters are `X`'s](https://man7.org/linux/man-pages/man3/mkstemp.3.html):

>  The last six characters of template must be "XXXXXX" and these are replaced with a string that makes the filename unique. Since it will be modified, template must not be a string constant, but should be        declared as a character array.

The BSD call only asks for "some number of `Xs' appended to it".